content/blog/2023/2023-01-10-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,160 @@
+*** Container image deprecation for the blueocean container
+*** Removed the deprecated install-plugins.sh script from Docker images
+*** Exit (and restart) lifecycle change in the Docker images
+** ANTLR 2 to ANTLR 4 transition complete, Jenkins core compiles
+*** Thanks to Alex Earl and Basil Crow!
+*** Included in Jenkins 2.376
+* Platform documentation
+** Short https://www.jenkins.io/doc/administration/requirements/servlet-containers/#sidebar-content[guide] about web containers
+* Platform WIP
+** Experiments with RISC-V agents with JDK17/19/20
+** Experiments with Windows 2022
+
+== User Experience Update
+
+Contributed by: https://github.com/MarkEWaite[Mark Waite]
+
+Jenkins LTS and weekly releases in 2022 have included significant user experience improvements thanks to the work of Jan Faracik, Tim Jacomb, Alex Brandes, Daniel Beck, and many others. Table layouts, menu entries, icons, themes, breadcrumbs, and more have been updated to give Jenkins a new, fresh look in 2022.
+
+== Documentation Update
+
+Contributed by: https://github.com/kmartens27[Kevin Martens]
+
+This year, we saw documentation contributions from new and seasoned Jenkins users.
+These contributions included blog posts, documentation additions and updates, documentation migration, and other improvements.
+All of this combined has helped expand and empower the Jenkins community.
+
+The Jenkins project saw 48 blog posts, submitted by 23 different authors, and overall 814 PRs were merged.
+The contributions are a result of the community and collaboration, with various projects throughout the year such as She Code Africa, Google Summer of Code, and Hacktoberfest.
+
+=== Pipeline Steps Reference
+
+Thanks to the work of https://www.jenkins.io/blog/authors/vihaanthora/[Vihaan Thora], contributing via https://www.jenkins.io/blog/2022/10/10/pipeline-steps-improvement-gsoc-report/#project-specific-guidance[Google Summer of Code], the https://www.jenkins.io/doc/pipeline/steps/[Pipeline steps] documentation has been improved.
+The Pipeline Steps reference guide is invaluable for developers when working in Jenkins and utilizing plugins.
+The updates include search functionality, overall appearance, and a reduction in page load.
+
+image:/images/post-images/2023/01/10/jenkins-newsletter/image5.png[image,width=624,height=388]
+
+=== Algolia search
+
+image:/images/post-images/2023/01/10/jenkins-newsletter/image6.png[image,width=275,height=52]
+
+The Jenkins site search has been updated to use the latest version of https://www.algolia.com/[Algolia].
+We want to recognize and thank https://www.jenkins.io/blog/authors/halkeye/[Gavin Mogan] for all of his work on this and Algolia for donating the search functionality.
+The site search now provides more relevant results and suggestions for users.
+A visual update was included as part of the upgrade, resulting in the new look and UI.
+
+image:/images/post-images/2023/01/10/jenkins-newsletter/image7.png[image,width=363,height=317]
+
```

Comment:
```suggestion
=== Community and Stories sites

In 2022, the Jenkins project was also able to launch two new sites for community engagement and involvement.

* link:https://community.jenkins.io/[community.jenkins.io] now provides a space for community discourse and communication.

* link:https://stories.jenkins.io/[stories.jenkins.io] is a site dedicated to sharing the experiences and stories of Jenkins users, developers, contributors, or anyone else that Jenkins has impacted. 
```
There should be a note of the community and stories sites as they launched in 2022 as well.